### PR TITLE
Add support for step managed endpoint extension.

### PR DESCRIPTION
**Pull request by joshdrake**
Adds support for display information from the Smallstep Managed Endpoiint x509 extension.

💔Thank you!

### CODE REVIEW

File: certinfo.go [hunk 1]
```diff
@@ -84,6 +85,15 @@ var stepProvisionerType = [...]string{
 	"Nebula", // Type 11
 }
 
+// stepManagedEndpointKind are string representations of the managed endpoint kind (int)
+// in the step managed endpoint extension.
+var stepManagedEndpointKind = [...]string{
+	"NOOP",     // Type 0, is not supported
```
Comment (maraino) [inline certinfo.go:91]:
```suggestion
	"Unknown",     // Type 0, is not supported
```

File: certinfo.go [hunk 2]
```diff
@@ -1041,6 +1056,31 @@ func CertificateText(cert *x509.Certificate) (string, error) {
...
+				// Get kind name
+				var kind string
+				if len(stepManagedEndpointKind) > val.Kind {
+					kind = stepManagedEndpointKind[val.Kind]
+				} else {
+					kind = fmt.Sprintf("%d (unknown)", val.Kind)
+				}
+
+				buf.WriteString(fmt.Sprintf("%16sKind: %s\n", "", kind))
+				if len(val.EndpointID) != 0 {
```
Comment (maraino) [inline certinfo.go:1081]: What's the use case for this condition?

**Review by maraino — CHANGES_REQUESTED** [review]

File: certinfo.go (hunk 2 above)
Comment (maraino) [reply certinfo.go:1081]: If the sequence does not have an endpoint id is going to fail before.

Comment (joshdrake) [reply certinfo.go:1081]: Ah, I previously had additional elements in the sequence, fixing this.

Comment (joshdrake) [reply certinfo.go]: [61bf429](https://github.com/smallstep/certinfo/pull/35/commits/61bf42957fdc801d9ce15e910d7f486e5ecb600a)

File: certinfo.go [hunk 3]
```diff
@@ -1041,6 +1056,29 @@ func CertificateText(cert *x509.Certificate) (string, error) {
...
+				// Get kind name
+				var kind string
+				if len(stepManagedEndpointKind) > val.Kind {
+					kind = stepManagedEndpointKind[val.Kind]
+				} else {
+					kind = fmt.Sprintf("%d (unknown)", val.Kind)
+				}
+
+				buf.WriteString(fmt.Sprintf("%16sKind: %s\n", "", kind))
+				buf.WriteString(fmt.Sprintf("%16sEndpointID: %s\n", "", string(val.EndpointID)))
```
Comment (maraino) [inline certinfo.go:1081]:
```suggestion
				buf.WriteString(fmt.Sprintf("%16sEndpointID: %s\n", "", val.EndpointID))
```

**Review by maraino — APPROVED** [review]
I've fixed some minor lint issues. LGTM

**Pull request merged**